private/email/templates/orders/new.html

```diff
@@ -265,6 +265,8 @@
                                                       <td valign="middle" align="left">
                                                         {{#if variantImage}}
                                                           <img src="{{variantImage}}"  width="50" height="50" alt="" />
+                                                        {{else if productImage}}
+                                                          <img src="{{productImage}}"  width="50" height="50" alt="" />
```

Comment:
In some cases, the variantImage is not available, but a productImage is available, and can be used instead of falling to the default.